cmake/projects/libuv/ci/matrix.json

```diff
@@ -0,0 +1,18 @@
+[
+{ "example": "libuv", "toolchain": "clang-cxx17",                                     "os": "ubuntu-20.04", "python": "3.8", "script": "build.sh"  },
```

Comment:
why a special matrix.json file? Do you need something installed? It isn't needed to get all CI-jobs green, some are allowed to fail. The failing ones will be recorded in the [Per package testing table](https://cpp-pm.github.io/hunter/) (link from Readme.md)

Reply:
this is for libuv not realy needed i changed that.